### PR TITLE
Please review Hashing-1

### PR DESCRIPTION
**Pull request by arzoouzma01**

### CODE REVIEW

File: Problem_1.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
+/*
+   
+Given an array of strings, group anagrams together.
+
+Example: Input: ["eat", "tea", "tan", "ate", "nat", "bat"], Output: [ ["ate","eat","tea"], ["nat","tan"], ["bat"] ]
+
+Note: All inputs will be in lowercase. The order of your output does not matter.
+*/
+
+//Time comlpexity - O(nk)
```
Comment (super30admin) [inline Problem_1.java:10]: O(nKlogk) because we are sorting as well

File: Problem_3.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
+/*
+   
+word pattern
+*/
+
+// Time Complexity :
+// Space Complexity :
+// Did this code successfully run on Leetcode :
```
Comment (super30admin) [inline Problem_3.java:8]: Missing items here....

File: Problem_2.java [hunk 3]
```diff
@@ -0,0 +1,41 @@
+/*
+   
+Isomorphic strings
+*/
+
+// Time Complexity :  O(n)
+// Space Complexity : O(n)
```
Comment (super30admin) [inline Problem_2.java:7]: O(1)....as discussed in class

File: Problem_2.java [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+        { 
+          char ch1=t.charAt(i); 
+          char ch2=s.charAt(i);
+          if(map.containsKey(ch1))  
+          {
+            char temp=map.get(ch1);
+             if(temp!=ch2) 
+              return false; 
+          }
+          else if(map.containsValue(ch2))
```
Comment (super30admin) [inline Problem_2.java:29]: This is O(n) and makes the solution O(n2) inefficient. Will result in interview rejection

**Comment by super30admin** [comment]
Kindly review inline comments. A better job is expected

**Comment by arzoouzma01** [comment]
Closing old request

**Pull request closed without merging**